addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -2414,22 +2414,19 @@ export class OdooEditor extends EventTarget {
         if (this._isResizingTable || !traversedNodes.some(node => !!closestElement(node, 'td') && !closestElement(node, '[data-oe-protected="true"]'))) {
             return false;
         }
-        let range;
-        if (selection.rangeCount > 1) {
-            // Firefox selection in table works with multiple ranges.
-            const startRange = getDeepRange(this.editable, {range: selection.getRangeAt(0)});
-            const endRange = getDeepRange(this.editable, {range: selection.getRangeAt(selection.rangeCount - 1)});
-            range = this.document.createRange();
-            range.setStart(startRange.startContainer, 0);
-            range.setEnd(endRange.startContainer, 0);
-        } else {
-            // We need the triple click correction only for a bug in firefox
-            // where it gives a selection of a full cell as tr 0 tr 1. The
-            // correction makes it so it gives us the cell and not its neighbor.
-            // In all other cases we don't want to make that correction so as to
-            // avoid flicker when hovering borders.
-            range = getDeepRange(this.editable, { correctTripleClick: anchorNode && anchorNode.nodeName === 'TR' });
+        if (this.isFirefox) {
+            if (selection.rangeCount > 1) {
+                this.historyResetLatestComputedSelection(true);
+            } else if(ev && closestElement(ev.target, 'table') === closestElement(selection.anchorNode, 'table')) {
```

Comment:
What is the idea with this condition?

```suggestion
            } else if (ev && closestElement(ev.target, 'table') === closestElement(selection.anchorNode, 'table')) {
```

Reply:
I added this `else if` statement because when I set the selection in the `if` block, the selection stops getting updated even if my cursor moves to the next `td`. So, what I'm doing is setting the focus node at `ev.target`, ensuring that when I move the cursor to other `tds`, the selection gets updated accordingly.

As for the conditions, firstly, I'm checking if the event is available, as I've observed that only `mousemove` passes the event when calling this function. This check is necessary because we only need to set the selection when selecting the table with the mouse. and I've included the second condition because in Firefox, when selecting with the mouse, it only selects within that table, and if the cursor goes out of that table, it stops updating the selection at that point.